opm/simulators/timestepping/AdaptiveTimeStepping_impl.hpp

```diff
@@ -26,6 +26,7 @@
 #include <stdexcept>
 
 #include <fmt/format.h>
+#include <fmt/ranges.h>
```

Comment:
Mmh, not introduced here but we now have a hard dependency of fmt if someone uses the module in its own application.